src/java/org/apache/cassandra/io/sstable/format/SSTableFormat.java

```diff
@@ -43,7 +43,8 @@
     public static enum Type
     {
         //The original sstable format
-        BIG("big", BigFormat.instance);
+        BIG("big", BigFormat.instance),
+        V3("bigv3", BigFormatV3.instance);
```

Comment:
Well, this is not a different format, it is a different version

Reply:
Correct.  I think you need to better document the concept which is to "Freeze" the format from the last major release.  Similar to how C* supports the last major Messaging Version to support live upgrade and downgrade.